docs/installation/debian.rst

```diff
@@ -42,11 +42,18 @@ Install from apt.mopidy.com
 
 #. Add the archive's GPG key::
 
-       wget -q -O - https://apt.mopidy.com/mopidy.gpg | sudo apt-key add -
+       sudo mkdir -p /usr/local/share/keyrings
+       wget -q -O - https://apt.mopidy.com/mopidy.gpg | gpg --dearmor | \
+         sudo tee /usr/local/share/keyrings/mopidy-archive-keyring.gpg
 
 #. Add the APT repo to your package sources::
 
-       sudo wget -q -O /etc/apt/sources.list.d/mopidy.list https://apt.mopidy.com/buster.list
+       echo "\
+       deb [signed-by=/usr/local/share/keyrings/mopidy-archive-keyring.gpg] \
+       https://apt.mopidy.com/ buster main contrib non-free
```

Comment:
Is this line supposed to have a trailing backslash?

Reply:
No, because I want a new-line before the `deb-src` definition that follows.